test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,78 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const userUrl = 'https://api.github.com/gists';
+const token = process.env.ACCESS_TOKEN;
+const gistUrl = 'https://api.github.com/gists';
+const usergistUrl = 'https://api.github.com/users/Maosorio92/gists';
+
+const axiosClient = axios.create({
+  headers: {
+    Authorization: `token ${token}`
+  }
+});
+describe('DELETE Method', () => {
+  let userResponse;
+  it('Creating GIST', async () => {
+    userResponse = await axiosClient.post(`${userUrl}`, {
```

Comment:
I recommend to better define this gist as a variable, so you can use it here and re-use it on the assert, instead of copying it 2 times

Reply:
Also, the variable 'userResponse' can be used on other tests to take profit of Hypermedia, but it is being defined inside this test, and tests should be independent. To solve this, we take proffit on mocha hooks: